pilot/proxy/envoy/testdata/lds-v0-egress-rule-tcp-auth-optin.json.golden

```diff
@@ -0,0 +1,845 @@
+{
+  "listeners": [
+   {
+    "address": "tcp://0.0.0.0:15001",
+    "name": "virtual",
+    "filters": [],
+    "bind_to_port": true,
+    "use_original_dst": true
+   },
+   {
+    "address": "tcp://0.0.0.0:443",
+    "name": "http_0.0.0.0_443",
+    "filters": [
```

Comment:
Who is still generating a http on 443? Can you remove this file and simply add a listener test for some other random port (9999)? And once that works, can you track down the http for https ports in a separate PR and remove that ?